src/swarm/node/model/NeoNode.d

```diff
@@ -726,6 +726,14 @@ public class NodeBase ( ConnHandler : ISwarmConnectionHandler ) : INodeBase
     private NeoListener neo_listener;
 
 
+    /***************************************************************************
+
+        Unix socket command handlers map.
+
+    ***************************************************************************/
+
+    protected BasicCommandHandler.Handler[istring] unix_socket_handlers;
```

Comment:
This could be an method, but that might be too much.

Reply:
To prevent derived classes from modifying the reference? Yeah... I guess that's not the standard approach in swarm.

Reply:
Yeah I'm fine with this as well.